packages/cf-component-text/src/TextTheme.js

```diff
@@ -0,0 +1,19 @@
+export default () => ({
+  'fontSize--normal': '1em',
+  'fontWeight--normal': 400,
+  'fontSize--normal': '.8em',
+  'lineHeight--small': 1.3,
+  'textAlign--start': 'left',
+  'textAlign--center': 'center',
+  'textAlign--justify': 'justify',
+  'textAlign--end': 'right',
+  'textTransform--capitalize': 'capitalize',
+  'textTransform--capitalize:first-word': 'capitalize',
+  'textTransform--lowercase': 'lowercase',
+  'textTransform--uppercase': 'uppercase',
+  'color--info': '#00a9eb',
+  'color--success': '#68970f',
+  'color--warning': '#fca520',
+  'color--error': '#ff3100',
+  'color--muted': '#gray'
+});
```

Comment:
Yea, we should stay consistent and use camelCase everywhere. It's easy to do it now than later.